### PR TITLE
Implement test for calculate average

### LINKED ISSUE #16: feats: test calculate_average function

### PR DESCRIPTION
**Pull request by SonQBChau**

### CODE REVIEW

File: R/features_creator.R [hunk 1]
```diff
@@ -14,9 +14,29 @@
 #' other_column = c(10, 11, 12)
 #' )
 #' calculate_average(df, "week_payment")
-
+#' c(4, 5, 6)
```
Comment (NikitaShymberg) [inline R/features_creator.R]: Nitpick: Could you delete this?

**Review by NikitaShymberg — CHANGES_REQUESTED** [review]
The tests failed for me because (I think) `get_matching_column_names` is not implemented. Could you `git rebase` or `git merge` main into your branch so that the tests pass?

File: R/features_creator.R (hunk 1 above)
Comment (SonQBChau) [reply R/features_creator.R]: done

**Comment by SonQBChau** [comment]
I just merged the main again, there error should go away

**Comment by NikitaShymberg** [comment]
Thanks Son, I've reviewed this and everything looks good now.

I had to make some small changes to make `check()` pass. I removed the definition of `calculate_average()` from `features_creator.R` (though I thought that the diff said that you did this but that's not what it looked like when I pulled the branch, idk what happened). I had to change some `None`s to `NULL`s from Rakesh's function definitions as well.

Anyway, everything works feel free to merge!

**Review by NikitaShymberg — APPROVED** [review]

**Pull request merged**